cli/execute/execute.go

```diff
@@ -0,0 +1,139 @@
+package execute
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/cli/arg"
+	"github.com/buildbuddy-io/buildbuddy/cli/log"
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/util/rexec"
+	"github.com/buildbuddy-io/buildbuddy/server/real_environment"
+	"github.com/buildbuddy-io/buildbuddy/server/util/flag"
+	"github.com/buildbuddy-io/buildbuddy/server/util/grpc_client"
+	"github.com/buildbuddy-io/buildbuddy/server/util/mdutil"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+	"google.golang.org/grpc/metadata"
+
+	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
+	bspb "google.golang.org/genproto/googleapis/bytestream"
+)
+
+var flags = flag.NewFlagSet("execute", flag.ContinueOnError)
+
+// Bazel-equivalent flags.
+var (
+	target        = flags.String("remote_executor", "grpcs://remote.buildbuddy.io", "Remote execution service target.")
+	instanceName  = flags.String("remote_instance_name", "", "Value to pass as an instance_name in the remote execution API.")
+	timeout       = flags.Duration("remote_timeout", 1*time.Hour, "Timeout used for the action.")
+	remoteHeaders = flag.New(flags, "remote_header", []string{}, "Header to be applied to all outgoing gRPC requests, as a `NAME=VALUE` pair. Can be specified more than once.")
```

Comment:
Could we wrap the FlagSet struct and add the additional flag types there instead of exposing `New`? It would be nice if flag usage was consistent between CLI and server.

Reply:
was considering that, but got lazy. Will send out a separate PR for that shortly

Reply:
OK actually it's not so simple, because `Slice()` and `Struct()` have type parameters, and IIRC struct methods can't have per-method type params like that.

Would prefer to stick with this for now.